test/libsolidity/semanticTests/array/byte_array_storage_layout.sol

```diff
@@ -41,17 +41,17 @@ contract c {
 // ====
 // compileViaYul: also
 // ----
-// storage: empty
+// storage_empty -> true
```

Comment:
The auto-updater uses `true` here and not `1`?

Reply:
Yes, looks like that the auto-updater uses `true` here.

Reply:
Ah ok, it did not work as expected. To test this I just changed `true` to `false` and run the update, the got changed to `true`. But if the tests consists of `0` or `1` the update will change it to `0` or `1`.

Reply:
Then let's just use 0 and 1 everywhere.

Reply:
oki